lib/main.dart

```diff
@@ -76,19 +79,20 @@ class _RandomWordsState extends State<RandomWords> {
   }
 
   Widget _buildRow(WordPair pair) {
-    final alreadySaved = _saved.contains(pair);
+    final saved = _saved.contains(pair);
```

Comment:
What is the point of renaming this variable? It gives us information on whether some startup name is already saved or not. I suppose you wanted to make it more understandable, but actually, now it brings even less information than the first version (alreadySaved). 

Consider renaming it to something like `savedPair`.

Reply:
This happened due to my attempt to add other functionality. Fixed)